### ISSUE TITLE
Missing web interface

### ISSUE THREAD

**Issue by pascalgoty** [ejurgensen/forked-daapd#480]
It's source built and i'm getting a 404 not found when i'm browsing http://ip:3689. I've search everywhere and there's no "webface" directory resulting from the build. I'm only getting a prompt for a user / pass (admin/unused) and after that i got the 404 and the webface missing.

here's the config i've used : (it's dockerize)
/configure \
	--build=$CBUILD \
	--enable-chromecast \
	--enable-itunes \
	--enable-lastfm \
	--enable-mpd \
	--host=$CHOST \
	--infodir=/usr/share/info \
	--localstatedir=/var \
	--mandir=/usr/share/man \
	--prefix=/app \
	--sysconfdir=/etc && \

[  LOG]     main: Forked Media Server Version 25.0 taking off
[  LOG]     main: Built Jan 14 2018 with: --enable-itunes --enable-lastfm --enable-chromecast --enable-mpd --enable-verification --with-alsa
[  LOG]     main: mDNS init
[  LOG]     mdns: Avahi state change: Client registering
[  LOG]       db: Now vacuuming database, this may take some time...
Server startup complete. Host name is datastor.local. Local service cookie is 127322544.
[  LOG]       db: Database OK with 0 active files and 6 active playlists
Service "datastor" (/etc/avahi/services/ssh.service) successfully established.
Service "datastor" (/etc/avahi/services/sftp-ssh.service) successfully established.
[  LOG]     mdns: Avahi state change: Client running
[  LOG]     scan: Bulk library scan completed in 0 sec (with file scan disabled)
[  LOG]      lib: Library init scan completed in 0 sec
[  LOG]    cache: Beginning DAAP cache update
[  LOG]    cache: DAAP cache updated
[  LOG]    httpd: Could not lstat() /app/share/forked-daapd/webface/: No such file or directory

**Comment by ejurgensen** [comment]
Are you building from a 25.0 release tarball? The web interface isn't included there, because it was added after the 25.0 release. If you clone from github and try to build again you should get the web interface.

**Comment by pascalgoty** [comment]
Wow.. i'm stupid. Thanks 👍

**Comment by sanjayradia** [comment]
I am running a dockerized version of forked-daapd:
[2018-02-21 02:07:43] [  LOG]     main: Forked Media Server Version 25.0 taking off
[2018-02-21 02:07:43] [  LOG]     main: Built Feb 16 2018 with: --enable-itunes --enable-lastfm --enable-chromecast --enable-mpd --enable-verification --with-alsa

I got it from docker hub https://hub.docker.com/r/linuxserver/daapd/
I get the error message: [ LOG] httpd: Could not lstat() /app/share/forked-daapd/webface/: No such file or directory

Looks like it was built in Feb 2018. The path is indeeded missing -- is this merely a config issue

**Comment by ejurgensen** [comment]
Err, did you read the above?

You might also be interested in https://github.com/linuxserver/docker-daapd/issues/16

**Comment by 6502c** [comment]
I did a fresh install of forked-daapd on my Raspberry Pi 4.
The version that was installed is 26.4.
I got everything setup and running so I can listen to my music library with forked-daapd on my Pinnacle Soundbridge.
But the web interface for administration of forked-daapd isn't available. The folder `/usr/share/forked-daapd/htdocs/` is empty.
What would be the recommended way to also get the admin web interface working in this situation?
